README.md

```diff
@@ -113,6 +113,33 @@ RUN dpkg --add-architecture arm64 && \
 $ docker build -t my/image:tag path/to/where/the/Dockerfile/resides
 ```
 
+### Passing environment variables into the build environemnt
+
+By default, `cross` does not pass any environment variables into the build
+environment from the calling shell. Sometimes this is really useful, for
```

Comment:
Minor nitpick, but "this" on this line seems to refer to the fact that `cross` *doesn't* pass through environment variables by default when I think you want it to mean the opposite. Maybe instead say "This is chosen as a safe default as most use cases will not want the calling environment leaking into the inner execution environment. But in the instances that you do want to pass through environment variables, this can be done through the use of a whitelist."